developer-docs-site/docs/guides/system-integrators-guide.md

```diff
@@ -3,49 +3,47 @@ title: "System Integrators Guide"
 slug: "system-integrators-guide"
 ---
 
-:::tip 
-This documentation is currently under construction with more being added on a regular basis.
-:::
-
-If you provide blockchain services to your customers and wish to add the Aptos blockchain to your platform, then this guide is for you. This system integrators guide will walk you through all you need to integrate the Aptos blockchain into your platform. This guide assumes that you are familiar with the blockchains.
+If you provide blockchain services to your customers and wish to add the Aptos blockchain to your platform, then this guide is for you. This system integrators guide will walk you through all you need to integrate the Aptos blockchain into your platform.
 
 ## Overview
 
-This guide will overview the following topics for integrating with Aptos:
-- Preparing an environment for testing.
-- Create an account on the blockchain.
-- Exchange account identifiers with another entity on the blockchain, for example, to perform swaps.
-- Create a transaction.
-- Obtain a gas estimate and validate the transaction for correctness.
-- Submit the transaction to the blockchain.
-- Wait for the outcome of the transaction.
-- Query historical transactions and interactions for a given account with a specific account, i.e., withdraws and deposits.
+This document will guide you through the following tasks to integrate with Aptos:
+1. Prepare an environment for testing.
+1. Create an account on the blockchain.
+1. Exchange account identifiers with another entity on the blockchain, for example, to perform swaps.
+1. Create a transaction.
+1. Obtain a gas estimate and validate the transaction for correctness.
+1. Submit the transaction to the blockchain.
+1. Wait for the outcome of the transaction.
+1. Query historical transactions and interactions for a given account with a specific account, i.e., withdraws and deposits.
+
+## Networks
 
-## Getting Started
+There are three well-supported networks for integrating with the Aptos blockchain:
 
-There are two well-supported approaches for integrating with the Aptos blockchain:
+1. Local testnet -- our standalone tool for local development against a known version of the codebase with no live network.
```

Comment:
I mean, I guess it's "live".  I would maybe say... `with your own blockchain`

Reply:
I used "external" instead.